invenio_accounts/ext.py

```diff
@@ -93,6 +93,11 @@ def monkey_patch_flask_security():
             recoverable.encrypt_password = encrypt_password
             registerable.encrypt_password = encrypt_password
 
+        # disable remember me cookie generation
```

Comment:
Since the tests are failing, perhaps expand the comment a little (so that we can remember why it was disabled when we all are not around anymore :-):

```
Disable remember me cookie generation as it does not work with
session activity tracking (a remember me token will bypass revoking 
of  a session).
```